app/views/templates/_form.html.erb

```diff
@@ -40,9 +40,8 @@
       <% end -%>
 
       <div class='form-group'>
-        <div class="col-md-12">
-            <div id="react-ace"></div>
-              <%= mount_react_component('Editor', "#react-ace", {
+        <div id="react-ace" class="col-md-12">
```

Comment:
Can you please move this div to the `Editor` component? 
this id is used in a few integration tests only for this component

Reply:
I moved this div to the Editor component. Let me know if what I did is what you meant because I wasn't sure.